grafana-plugin/src/pages/insights/Insights.tsx

```diff
@@ -39,154 +39,184 @@ import getTotalAlertGroupsScene from './scenes/TotalAlertGroups';
 import getTotalAlertGroupsByStateScene from './scenes/TotalAlertGroupsByState';
 import getVariables from './variables';
 
+const getDefaultStackValue = (isOpenSource: boolean) =>
+  isOpenSource ? 'self_hosted_stack' : location.host.split('.')[0];
```

Comment:
is it the only way to get stack? 🧐

Reply:
Julia is going to expose it as part of `/organization` endpoint and then we will switch to it. Till it's ready we need to parse url